ebean-core/src/main/java/io/ebeaninternal/server/deploy/BeanProperty.java

```diff
@@ -1160,7 +1160,26 @@ public boolean isFetchEager() {
   public boolean isLob() {
     return lob;
   }
-
+  
+  /**
+   * Returns true if this <code>isLob()</code> or the type will effectively map to a lob. 
+   */
+  public boolean isDbLob() {
+    if (lob) {
+      return true;
+    }
+    switch (dbType) {
+    case DbPlatformType.JSON:
+    case DbPlatformType.JSONB:
+      return dbLength == 0; // must be analog to DbPlatformTypeMapping.lookup
+    case DbPlatformType.JSONBlob:
+    case DbPlatformType.JSONClob:
+      return true;
```

Comment:
Should we delegate that decision to the DB-platform?

Reply:
At the moment I think the above is good and that we should not delegate this to the DB-platform.  I **_believe_** JSON types are generally more "clob" than "varchar" in nature ... it is a little tempting to not even have the dbLength == 0 check.